_rules/form-fields-not-completed-334972.md

```diff
@@ -0,0 +1,227 @@
+---
+id: 334972
+name: Required form fields not completed
+rule_type: atomic
+description: |
+  This rule checks that text error messages identify required form fields that were not completed.
+accessibility_requirements:
```

Comment:
It seems to me that this by itself pretty much is a failure of 3.3.1. Baring better definitions, I don't think this needs to be part of a composite rule.